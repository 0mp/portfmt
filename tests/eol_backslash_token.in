PLIST_FILES=	bin/blabla\
		bin/abc
