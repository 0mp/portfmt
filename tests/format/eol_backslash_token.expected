PLIST_FILES=	bin/abc \
		bin/blabla
